librz/include/rz_endian.h

```diff
@@ -127,6 +127,28 @@ static inline void rz_write_at_be64(void *dest, ut64 val, size_t offset) {
 	rz_write_be64(d, val);
 }
 
+static inline ut128 rz_read_be128(const void *src) {
```

Comment:
This could go together with `RzBuf` and `RzVector` changes PR in a separate commit